modules/installation-complete-user-infra.adoc

```diff
@@ -149,12 +149,17 @@ cluster machines.
 ifndef::ibm-power[]
 . For an installation with Fibre Channel Protocol (FCP), additional steps are required to enable multipathing. Do not enable multipathing during installation.
 +
-See "Enabling multipathing with kernel arguments on RHCOS" in the _Post-installation configuration_ documentation for more information.
+[NOTE]
+====
+When you install with multiple paths, you must enable multipathing directly after the installation, not at a later point in time, as this can cause problems.
```

Comment:
As per the previous comment, multipathing ideally would be turned on even before you do `coreos-installer install`, so it'd be more accurate to say "at installation time" instead of "after the installation". Maybe something like

```suggestion
When installing with multipath, it is strongly recommended to enable it at installation time, not at a later point in time, as this can cause problems.
```

?